com.chartboost.mediation/Runtime/Banner/ChartboostMediationBannerIOS.cs

```diff
@@ -14,7 +17,21 @@ public sealed class ChartboostMediationBannerIOS : ChartboostMediationBannerBase
         public ChartboostMediationBannerIOS(string placement, ChartboostMediationBannerAdSize size) : base(placement, size)
         {
             LogTag = "ChartboostMediation Banner (iOS)";
-            _uniqueId = _chartboostMediationGetBannerAd(placement, (int)size);
+
+            if (size.Name == "ADAPTIVE")
+            {
+                Logger.LogError(LogTag,$"Adaptive sizes are not supported for `ChartboostMediationBannerAd`. Use `ChartboostMediationBannerView` instead");
+                return;
```

Comment:
This logic seems repeated for both Android/iOS, could you put it into a utility method?

```csharp
public float? GetFixedSize(string name)
{
    if (name == "ADAPTIVE")
    {
        Logger.LogError(LogTag,$"Adaptive sizes are not supported for `ChartboostMediationBannerAd`. Use `ChartboostMediationBannerView` instead");
        return null;
    }

    var fixedSize = Name switch
    {
        "STANDARD" => 0,
        "MEDIUM" => 1,
        "LEADERBOARD" => 2,
        _ => 0
    };
    return fixedSize;
}
```

If you use enums it would be even better as described in my previous comment. Also, you could make this method part of `ChartboostMediationBannerAdSize` or create a extension method. Whatever works best.